svg/elements/feComposite.json

```diff
@@ -24,19 +24,19 @@
               "version_added": null
             },
             "opera": {
-              "version_added": false
+              "version_added": "15"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "14"
             },
             "safari": {
-              "version_added": null
+              "version_added": true
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": true
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "1.5"
```

Comment:
Chrome has full support from version 1. It got Blink in v28 which corresponds to Samsung Internet 1.5